govcd/cse.go

```diff
@@ -233,8 +233,8 @@ func (cluster *CseKubernetesCluster) GetSupportedUpgrades(refreshOvas bool) ([]*
 		if err != nil {
 			continue // This means it's not a TKGm OVA, or it is not supported, so we skip it
 		}
-		// The OVA can be used if the TKG version is higher than the actual and the Kubernetes version is at most 1 minor higher.
-		if targetVersions.compareTkgVersion(cluster.TkgVersion.String()) == 1 && targetVersions.kubernetesVersionIsOneMinorHigher(cluster.KubernetesVersion.String()) {
+		// The OVA can be used if the TKG version is higher or equal than the actual and the Kubernetes version is at most 1 minor higher.
```

Comment:
```suggestion
		// The OVA can be used if the TKG version is equal to the actual or higher and the Kubernetes version is at most 1 minor higher.
```
"equal than" doesn't feel right

Reply:
Fixed, thanks!